python/graphstorm/model/gnn.py

```diff
@@ -874,6 +895,7 @@ def do_mini_batch_inference(model, data, batch_size=1024,
                                                    task_tracker=task_tracker,
                                                    feat_field=data.node_feat_field,
                                                    target_ntypes=infer_ntypes)
+        model.eval()
```

Comment:
why do you call eval here?

Reply:
embeddings = model.normalize_node_embs(embeddings) should be called with model.eval()